doc/template.html

```diff
@@ -44,6 +44,18 @@
   </div>
   <script src="assets/sh_main.js"></script>
   <script src="assets/sh_javascript.min.js"></script>
-  <script>highlight(undefined, undefined, 'pre');</script>
+  <script>
+    highlight(undefined, undefined, 'pre');
+
+    // stop mousewheel scrolling on the sidebar from affecting the main column
+    document.getElementById('column2').addEventListener('wheel', function(e) {
+      if (e.deltaMode !== 0) return;
+      if ((e.deltaY > 0 && this.scrollHeight - this.offsetHeight - this.scrollTop <= 0) ||
+          (e.deltaY < 0 && -e.deltaY >= this.scrollTop)) {
+        this.scrollTop = e.deltaY > 0 ? this.scrollHeight : 0;
+        e.preventDefault();
+      }
+    });
```

Comment:
If you go this route, may I suggest to throttle the listener to avoid horrible perfs janks?